### ISSUE TITLE
feedback on hidden articles in docs

### ISSUE THREAD

**Issue by samuba** [brillout/telefunc#73]
Hi again, 
when I first found telefunc yesterday I was not sure about it's approach and if it would fit my needs because the start page makes for a good selling pitch but I needed to read through a lot of the docs pages to stitch together how all of this would feel like in actual code, what architectural choices have been made, edge cases, and if the "philosophy" or way telefunc approaches things actually fits my style.
So I was very surprised when I discovered the **hidden** [RPC](https://telefunc.com/RPC) article which does a good job in explaining most of these things. Im pretty sure my adoption journey would have been smoother if I would have found that article in the beginning.

So I got curious and looked in the docs code to see what else was hidden from sight and discovered the other articles which after reading through them give me more confidence in the whole approach and how the project is steered.
That being said I really do not understand why these articles are hidden. I think hiding them is a unnecessary hurdle to adoption.I would just put them in the left side nav to make them discoverable.

Of course this is all very subjective but I bet there are others like me : )

**Comment by brillout** [comment]
Thanks 💚. It's actually very usefull to have a glimpse of how you (would have liked to) approach Telefunc.

I think you're right.

FYI I used to [oversell the theory behind RPC](https://github.com/brillout/wildcard-api#learning-material). Having failed to convince folks, I went for a more practical approach. But I think you're right, both the theory and a practical approach are important.

PR welcome to dig out the hidden docs you believe should be exposed &mdash; if that's something you'd be up for.
